src/Analyzers/CSharp/Analyzers/AddRequiredParentheses/CSharpAddRequiredExpressionParenthesesDiagnosticAnalyzer.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable enable
```

Comment:
This is leaving two blank lines, which is probably going to anger some analyzer.

Reply:
Yeah it’s really upset the analyzers. Going to rerun automation and fix this